platooning_control_ihp/include/platoon_control_ihp_config.h

```diff
@@ -36,19 +36,37 @@ struct PlatoonControlIHPPluginConfig
   int cmdTmestamp = 100;
   double integratorMax = 100;
   double integratorMin = -100;
-  double Kdd = 4.5;                             //coefficient for smooth steering
+  double Kdd = 4.5;                             //coeficient for smooth steering
   double wheelBase = 3.09;
   double lowpassGain = 0.5;
   double lookaheadRatio = 2.0;
   double minLookaheadDist = 6.0;
   std::string vehicleID = "DEFAULT_VEHICLE_ID";         // Vehicle id is the license plate of the vehicle
-  int     shutdownTimeout = 200;                // ms 
-  int     ignoreInitialInputs = 0;              // num inputs to throw away after startup
+  // added for gap regulation
+  double vehicleLength = 5.0;     // m
+
+  // UCLA: Added for IHP control 
+  // ---------------------- UCLA: parameters for IHP platoon trajectory regulation ----------------
+  /**
+  * \brief Parameter sets for IHP platoon trajectory regulation algorithm. 
+  * Please refer to the updated design doc for detailed parameter description.
+  */
+  double ss_theta   = 4.0;    // Stanstill determining threshold, in m/s.
+  double standstill = 2.0;    // Stanstill reaction time adjuster, in s.
+  double inter_tau  = 1.5;    // Inter-platoon time gap, refer to bumper to bumper gap time, in s.
+  double intra_tau  = 0.6;    // Intra-platoon time gao, refer to bumper to bumper gap time, in s.
+  double gap_weight = 0.9;    // Weighted ratio for time-gap based calculation, unitless.
+  double time_step  = 5;      // The time step ga regulation algorithm uses to calculate desired position, in s.
+  bool   test_front_join = false;   //Flag to enable/disable front join functionality with two vehicles.
+                                    // Flag can be set to true, to test front join functionality with two vehicles
+                                    // But in normal operating conditions it should be set to false
+  //------------------------------------------------------------------------------------------------
+
   
   
-  friend std::ostream& operator<<(std::ostream& output, const PlatoonControlIHPPluginConfig& c)
+  friend std::ostream& operator<<(std::ostream& output, const PlatooningControlIHPPluginConfig& c)
   {
-    output << "PlatoonControlIHPPluginConfig { " << std::endl
+    output << "InLaneCruisingPluginConfig { " << std::endl
```

Comment:
Name is totally changed!  Please be careful that you are working with the latest code base.  It's a good idea to scan over a PR page after you create it to be sure everything is in order.